### PR TITLE
story ui

### PR DESCRIPTION
**Pull request by njason**
sample config

```
{
  "mapboxAccessToken": "xxxx",
  "latLng": [40.689656366699694, -74.04497587916974],
  "zoom": 13,
  "layers": [
    {
      "id": "afternoon",
      "name": "Afternoon",
      "url": "layers/afternoon.tiff",
      "type": "raster"
    }
  ],
  "stories": [
    {
      "title": "Hot in the Heights",
      "content": "Its very hot in the heights",
      "mapState": {
        "lat": 40.745493,
        "lng": -74.043362,
        "zoom": 15
      }
    }
  ]
}
```

### CODE REVIEW

File: src/components/Story.svelte [hunk 1]
```diff
@@ -1,60 +1,37 @@
...
-            input$aria-label=""
-            color="secondary"
-        />
-    </Content>
-</Paper>
-{/if}
+<div class="accordion-container">
+    <Accordion>
+        {#each config.stories as story}
+            <Panel on:click={() => setMapState(story.mapState)}>
```
Comment (tzinckgraf-griid) [inline src/components/Story.svelte:31]: Does this get called on opening and closing the panel?

File: src/components/Story.svelte [hunk 2]
```diff
@@ -1,60 +1,37 @@
...
+    const unsubscribe = mapStore.subscribe((newMap: Map|null) => {
+        if (newMap != null) {
+            map = newMap;
+        }
+    });
+
+    onDestroy(unsubscribe);
+
+    function setMapState(mapState: MapState): void {
+        map?.flyTo(new LatLng(mapState.lat, mapState.lng), mapState.zoom);
```
Comment (tzinckgraf-griid) [inline src/components/Story.svelte:21]: Is the only map state just a zoom to a location?

File: src/components/Story.svelte (hunk 1 above)
Comment (tzinckgraf) [inline src/components/Story.svelte:31]: Does this get called on opening and closing?

Comment (tzinckgraf) [inline src/components/Story.svelte:31]: Imagine I am searching around on the map. I am looking at NYC, then I see this amazing story by njason about the heat in the heights. I click it, and I am zoomed into to somewhere NJ. I didn't want that. Now I am lost on the map.

Doesn't have to happen here, but maybe we should think about some sort of "back" button, or alternatively putting some map state ideas into the url, similar to google maps.

File: src/config.ts [hunk 3]
```diff
@@ -1,27 +1,39 @@
-export default interface Config {
-  mapboxAccessToken: string,
-  latLng: [number, number],
-  zoom: number,
-  layers: Layer[],
-  stories: Story[]
+import config from './configs/config.json';
```
Comment (tzinckgraf) [inline src/config.ts:1]: I like this. This is slick with the default import at the bottom.

**Pull request merged**